### PR TITLE
DWPF-1516 Minor cookie banner restyling

### PR DESCRIPTION
**Pull request by nicopicchio**

### CODE REVIEW

File: assets/stylesheets/components/_site_alert.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
     color: white;
     a, a:visited, a:link {
       color: white;
+      font-size: 1rem;
```
Comment (marcelkornblum) [inline assets/stylesheets/components/_site_alert.scss:11]: would it not work to instead make the class `govuk-body-s` in the banner?

File: assets/stylesheets/govuk-overrides/main.scss [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 .govuk-phase-banner {
   width: 100%;
+  border-bottom: none;
```
Comment (marcelkornblum) [inline assets/stylesheets/govuk-overrides/main.scss:23]: is it not the cookie banner with the border?

Comment (nicopicchio) [reply assets/stylesheets/govuk-overrides/main.scss:23]: no the cookie-banner is an external wrapper. what has the border is the phase banner which inside it https://github.com/uktrade/digital-workspace-v2/blob/6cf86a32444da0d7efa2153785f964e5867e9ddf/src/core/templates/tags/cookie_banner.html#L6

File: assets/stylesheets/components/_site_alert.scss (hunk 1 above)
Comment (nicopicchio) [reply assets/stylesheets/components/_site_alert.scss:11]: yeah good point, sorted it

**Review by marcelkornblum — APPROVED** [review]
merci!

**Pull request merged**